app/code/Magento/Catalog/Test/Mftf/ActionGroup/StorefrontAssertProductAttributeLabelVisibleActionGroup.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="StorefrontAssertProductAttributeLabelVisibleActionGroup">
```

Comment:
Please, correct the naming according to the naming convention (see section 11.3.5 of the [guidelines](https://devdocs.magento.com/guides/v2.4/coding-standards/technical-guidelines.html#113-functional-testing)).

Reply:
Done!